### ISSUE TITLE
iOS build fail on 0.6.2

### ISSUE THREAD

**Issue by cashdow** [LiquidPlayer/LiquidCore#139]
I did 'pod install' version 0.6.2 and  build, and I wanted to check if it was working.
but There were errors at build time in xcode.

I look around issue page but other person have no problems. But I didn't anything at code or something.. I just pod install and build.

xcode : 11.1
mac os : 10.14.6
ios build target : 12.0 and 11.0 either errors.
Here is my Podfile

platform :ios, '11.0'
   
   target 'liquid_test' do
     # Comment the next line if you don't want to use dynamic frameworks
     use_frameworks!
   platform :ios, '11.0'

target 'liquid_test' do
  # Comment the next line if you don't want to use dynamic frameworks
  use_frameworks!
  pod 'LiquidCore'
  # Pods for liquid_test

  target 'liquid_testTests' do
    inherit! :search_paths
    # Pods for testing
  end 

  target 'liquid_testUITests' do
    # Pods for testing
  end 
end

and Here is error 

Ld /Users/cashdow/Library/Developer/Xcode/DerivedData/liquid_test-cquaildrcquedpfvkjsiuksyshev/Build/Products/Debug-iphoneos/LiquidCore/LiquidCore.framework/LiquidCore normal arm64 (in target 'LiquidCore' from project 'Pods')
    cd /Users/cashdow/workspace_swift/liquid_test/Pods
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -target arm64-apple-ios11.0 -dynamiclib -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.1.sdk -L/Users/cashdow/Library/Developer/Xcode/DerivedData/liquid_test-cquaildrcquedpfvkjsiuksyshev/Build/Products/Debug-iphoneos/LiquidCore -L/Users/cashdow/workspace_swift/liquid_test/Pods/LiquidCore/deps/openssl-1.0.2o/lib-ios -F/Users/cashdow/Library/Developer/Xcode/DerivedData/liquid_test-cquaildrcquedpfvkjsiuksyshev/Build/Products/Debug-iphoneos/LiquidCore -filelist /Users/cashdow/Library/Developer/Xcode/DerivedData/liquid_test-cquaildrcquedpfvkjsiuksyshev/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/LiquidCore.build/Objects-normal/arm64/LiquidCore.LinkFileList -install_name @rpath/LiquidCore.framework/LiquidCore -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -dead_strip -Xlinker -object_path_lto -Xlinker /Users/cashdow/Library/Developer/Xcode/DerivedData/liquid_test-cquaildrcquedpfvkjsiuksyshev/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/LiquidCore.build/Objects-normal/arm64/LiquidCore_lto.o -Xlinker -export_dynamic -Xlinker -no_deduplicate -fembed-bitcode-marker -stdlib=libc++ -fobjc-arc -fobjc-link-runtime -lc++ -lcrypto -lssl -lz -framework JavaScriptCore -framework Foundation -framework JavaScriptCore -compatibility_version 1 -current_version 1 -Xlinker -dependency_info -Xlinker /Users/cashdow/Library/Developer/Xcode/DerivedData/liquid_test-cquaildrcquedpfvkjsiuksyshev/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/LiquidCore.build/Objects-normal/arm64/LiquidCore_dependency_info.dat -o /Users/cashdow/Library/Developer/Xcode/DerivedData/liquid_test-cquaildrcquedpfvkjsiuksyshev/Build/Products/Debug-iphoneos/LiquidCore/LiquidCore.framework/LiquidCore

Undefined symbols for architecture arm64:
  "node::MainSource(node::Environment*)", referenced from:
      node::LoadEnvironment(node::Environment*) in node.o
  "node::DefineJavaScript(node::Environment*, v8::Local<v8::Object>)", referenced from:
      node::Binding(v8::FunctionCallbackInfo<v8::Value> const&) in node.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by ericwlange** [comment]
Hi, can you try to use the latest checked-in version?  Apple has made a lot of changes lately with XCode, iOS and Catalina so it is hard to figure out what's breaking where.

```ruby
pod 'LiquidCore', git: 'https://github.com/LiquidPlayer/LiquidCore.git', commit: 'HEAD'
```

Let me know what issues you see.

**Comment by cashdow** [comment]
I did pod install above adress. and It was quite different error... 

In LiquidCore/opnessl/a_print.c 

#include "internal/cryptlib.h" file not found 

Am I something wrong?

**Comment by ericwlange** [comment]
This issue should be fixed in 0.7.2
